core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -622,24 +620,8 @@
     - cluster: default
       namespace: ci
       name: cluster-secrets-metal
-# azure
-- from:
-    pull-secret:
-      bw_item: quay.io
-      attachment: pull-secret
-    osServicePrincipal.json:
-      bw_item: os4-installer.openshift-ci.azure
-      attachment: osServicePrincipal.json
-    ssh-privatekey:
-      bw_item: jenkins-ci-iam
-      attachment: ssh-privatekey
-    ssh-publickey:
-      bw_item: jenkins-ci-iam
-      attachment: ssh-publickey
-  to:
-    - cluster: default
-      namespace: ci
-      name: cluster-secrets-azure
+# cluster-secrets-azure is not synced. Need to ask DPP team for this.
```

Comment:
`cluster-secrets-azure` has been out of synced?

```
$ oc get secret -n ci cluster-secrets-azure -o yaml | yq -r '.data | keys[]'
```

https://github.com/openshift/release/blob/ebec30a2ff2a099f3619bce4a239818f49a4c879/ci-operator/populate-secrets-from-bitwarden.sh#L287-L292

Reply:
This will be a question for Jim Minter -- can you send an email that clearly explains the difference between what's in the cluster and what's in BitWarden?

Reply:
Email sent to Jim.
Will do follow up on this when he replies.

Also for the `azure4`.

/hold cancel